Source/WebCore/loader/DocumentLoader.cpp

```diff
@@ -2109,6 +2114,25 @@ static bool canUseServiceWorkers(LocalFrame* frame)
 }
 #endif
 
+static bool shouldCancelLoadingAboutURL(const URL& url)
+{
+    if (!url.protocolIsAbout())
+        return false;
+
+    if (url.isAboutBlank() || url.isAboutSrcDoc())
+        return false;
+
+    if (!url.hasOpaquePath())
+        return false;
+
+#if PLATFORM(COCOA)
+    if (CocoaApplication::isInIWorkSuite() && !linkedOnOrAfterSDKWithBehavior(SDKAlignedBehavior::OnlyLoadWellKnownAboutURLs))
```

Comment:
Since we have had breakage in several other apps, I don’t think we should check for IWork. Just have a linked on after check.

Reply:
I didn't realize that. OK.